validator/accounts/v2/consts/consts.go

```diff
@@ -0,0 +1,22 @@
+package consts
```

Comment:
Why their own package?

Reply:
Thanks for pointing this out, I needed it for something to prevent a dependency loop but it's not needed anymore.